test.ts

```diff
@@ -780,6 +780,40 @@ Deno.test("outputToObjectArrayEmpty", function () {
   );
 });
 
+Deno.test("jsonFunctions", function () {
+  const db = new DB();
+
+  // The JSON1 functions should exist and we should be able to call them without unexpected errors
+  db.query(`SELECT json('["this is json"]')`);
+  db.query(`SELECT json('{"this is": "json"}')`);
+  db.query(`SELECT json('2')`);
```

Comment:
I think any one of these should be fine. We don't really need to unit test SQLite itself 😄

Reply:
done